image_get/worker/test/test_worker.py

```diff
@@ -0,0 +1,219 @@
+import json
+import pytest
+import asyncio
+import time
+import random
+import logging as log
+from worker.consumer import poll_consumer, consume
+from worker.util import process_image
+from PIL import Image
+from collections import deque
+from enum import Enum, auto
+from functools import partial
+
+log.basicConfig(level=log.DEBUG)
+
+
+class FakeMessage:
+    def __init__(self, value):
+        self.value = value
+
+
+class FakeConsumer:
+    def __init__(self):
+        self.messages = []
+
+    def insert(self, message):
+        self.messages.append(
+            FakeMessage(bytes(message, 'utf-8'))
+        )
+
+    def consume(self, block=True):
+        if self.messages:
+            return self.messages.pop()
+        else:
+            return None
+
```

Comment:
worker_1     | Traceback (most recent call last):
worker_1     |   File "worker/consumer.py", line 1, in <module>
worker_1     |     import worker.settings as settings
worker_1     | ModuleNotFoundError: No module named 'worker'
image_get_worker_1 exited with code 1